src/indexer/log_merge_policy.rs

```diff
@@ -6,26 +6,40 @@ use std::f64;
 const DEFAULT_LEVEL_LOG_SIZE: f64 = 0.75;
 const DEFAULT_MIN_LAYER_SIZE: u32 = 10_000;
 const DEFAULT_MIN_MERGE_SIZE: usize = 8;
+const DEFAULT_MAX_MERGE_SIZE: usize = 5000;
```

Comment:
That default is way too low. Let's make it `10_000_000` by default maybe?

Reply:
Sounds good to me